controller/include/pid.h

```diff
@@ -20,8 +20,10 @@ limitations under the License.
 #include "hal.h"
 #include "network_protocol.pb.h"
 
-inline constexpr AnalogPinId DPSENSOR_PIN = AnalogPinId::HAL_A0;
-inline constexpr PwmPinId BLOWERSPD_PIN = PwmPinId::PWM_3;
+inline constexpr AnalogPinId DPSENSOR_PIN = AnalogPinId::HAL_A5;
+inline constexpr PwmPinId BLOWERSPD_PIN = PwmPinId::PWM_6;
+
+// TODO(jlebar): Where did these numbers come from?
```

Comment:
The high one is not even used... Did you by chance reach out to whoever authored this and ask? If so and if that happened on an accessible place in Slack, link the thread here?

Reply:
I now understand that these are numbers in the range [0, 255] reflecting the pulse-width-modulation sent to the blower.  I think they're just dummy values that should be removed.  Updated the comment.